src/ddmd/ctfe/bc.d

```diff
@@ -2823,10 +2823,10 @@ const(BCValue) interpret_(const int[] byteCode, const BCValue[] args,
                             else
                             {
                                 // we will now resize the heap to 8 times of it's former size
-                                auto newHeap = new uint[](heapPtr.heapMax * 8);
+                                auto newHeap = new uint[](heapPtr.heapMax * 12);
```

Comment:
Above comment should be "12 times" to match.

Reply:
I included this one and my comments above in https://github.com/UplinkCoder/dmd/pull/3.